h2o-algos/src/test/java/hex/pca/PCAWideDataSetsTests.java

```diff
@@ -0,0 +1,404 @@
+package hex.pca;
+
+import hex.DataInfo;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import water.DKV;
+import water.Key;
+import water.Scope;
+import water.TestUtil;
+import water.fvec.Frame;
+
+import java.util.concurrent.ExecutionException;
+
+import static water.TestUtil.parse_test_file;
+
+/**
+	* Created by wendycwong on 2/27/17.
+	*/
+public class PCAWideDataSetsTests extends TestUtil {
+		public static final double TOLERANCE = 1e-6;
```

Comment:
8 character spacing? Use 2, it's Java. Everybody uses 2.
You can fix it all by clicking Code/ReformatCode in IntelliJ.